scripts/primir_quantification.py

```diff
@@ -45,9 +42,11 @@ def parse_arguments():
         type=Path
     )
     parser.add_argument(
-        '-s', '--sam',
-        help="Path to the SAM file containing the intersecting alignments.",
-        type=Path
+        '--collapsed',
+        help="Indicate that the file used in bedtools intersect has the\
+            reads collapsed by sequence and alignment. Default %(default)s.",
```

Comment:
It's better to use parentheses rather than `\` for splitting lines. For example:

```python
help=(
    "Line 1 continues"
    " on line 2."
)
```

Also, you need to indicate what's the expected format/syntax for collapsed reads (as produced by the FASTX collapser).